app/views/components/_figure.html.erb

```diff
@@ -9,4 +10,10 @@
       <%= caption %>
     </figcaption>
   <% end %>
+
+  <% if credit.present? %>
+    <figcredit class="app-c-figure__figcredit">
```

Comment:
I assume this wasn't meant to be custom element but rather part of the `<figcaption>` or a separate `<span>`?

Reply:
What is `figcaption`? We were trying to follow the pattern defined by captions, but weren't sure where this element is defined.

Reply:
Should we just make it a `span`?

Reply:
`<figcaption>` is a defined element representing a caption or a legend associated with an image/figure. They must be placed inside a ` <figure>` element to be semantically correct. There is no html element for credits though, so we can use a generic tag (i.e. `<span>` or we can include the credits inside the `<figcaption>` element.

Reply:
TIL! I thought figcaption was something we'd made up! 

I'll use `figcaption` for consistency.

Reply:
As @vanitabarrett mentioned, `<figure>` should only have one `<figcaption>` so it probably makes more sense to be included in the exiting tag.